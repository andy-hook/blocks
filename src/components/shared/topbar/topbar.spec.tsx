import React from "react"
import { renderWithTheme } from "@test-utils"

import Topbar from "./topbar"

describe("<Topbar />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <Topbar
        openMenu={jest.fn()}
        closeMenu={jest.fn}
        visible={true}
        open={true}
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
