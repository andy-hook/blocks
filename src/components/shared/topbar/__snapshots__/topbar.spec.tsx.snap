// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Topbar /> renders correctly 1`] = `
<div
  className="sc-gzVnrw hkfFDC"
  style={
    Object {
      "opacity": 0,
      "transform": "translate3d(0,-5rem,0)",
    }
  }
>
  <div
    className="sc-htoDjs clSpNh"
  >
    <div
      className="sc-iwsKbI kkVzut"
    >
      <div
        className="sc-gqjmRU heJvZy"
        style={
          Object {
            "opacity": 0,
            "transform": "translate3d(-5rem,0,0)",
          }
        }
      >
        <a
          className="sc-gZMcBi ijHXEV"
          href="/"
        >
          <div
            className="sc-EHOje foslyy"
          >
            <svg
              className="sc-bZQynM jYytzR"
            >
              <use
                xlinkHref="/icon-sprite.svg#blocks"
              />
            </svg>
          </div>
        </a>
      </div>
      <div
        style={
          Object {
            "transform": "translate3d(-5rem,0,0)",
          }
        }
      >
        <nav
          className="sc-bdVaJa bDWFJH sc-VigVT hucas"
        >
          <ul
            className="sc-bwzfXH iPhcok"
          >
            <li
              className="sc-htpNat iaJVDJ"
            >
              <a
                className="sc-bxivhb iUcbWs"
                href="/"
              >
                Latest blocks
              </a>
            </li>
            <li
              className="sc-htpNat iaJVDJ"
            >
              <a
                className="sc-ifAKCX kdWyHd"
                href="https://github.com/andy-hook/ten-blocks"
                onClick={[Function]}
                target="_blank"
              >
                View on Github
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
    <div
      className="sc-jTzLTM vrVLp"
    >
      <button
        className="sc-fjdhpX hsDKpW"
        onClick={[Function]}
      >
        <div
          className="sc-EHOje foslyy"
        >
          <svg
            className="sc-bZQynM jYytzR"
          >
            <use
              xlinkHref="/icon-sprite.svg#dark-mode"
            />
          </svg>
        </div>
      </button>
      <a
        className="sc-jzJRlG fwgwkd"
        href="/block/undefined"
        onClick={[Function]}
      >
        <div
          className="sc-EHOje foslyy"
        >
          <svg
            className="sc-bZQynM jYytzR"
          >
            <use
              xlinkHref="/icon-sprite.svg#shuffle"
            />
          </svg>
        </div>
      </a>
    </div>
    <div
      className="sc-dnqmqq jnoPwI"
      style={
        Object {
          "opacity": 0,
        }
      }
    />
  </div>
</div>
`;
